apps/nrm/views/partials/summary-table-your-report.html

```diff
@@ -378,6 +378,30 @@ <h2 class="details-header">{{#t}}pages.confirm.sections.your-report.header{{/t}}
     </tr>
     {{/values.is-this-the-first-chance-to-report}}
 
+    {{#values.when-last-contact}}
+    <tr>
+      <td>{{#t}}pages.confirm.fields.when-last-contact.label{{/t}}</td>
+      <td class="no-white-space">
+        <span>{{formattedwhenLastContact}}</span>
+      </td>
+      <td>
+        <a href="/nrm/when-last-contact/edit" class="button" aria-label="Change {{#t}}pages.confirm.fields.when-last-contact.label{{/t}}">Change</a>
```

Comment:
This may be out of scope of current change, but if there are translations used in the project, all elements should be covered.
In this particular case, the translation for the `Change` button should be defined in the buttons.json file.